examples/nyc_boros.py

```diff
@@ -1,36 +1,57 @@
 """
-Generate example images for GeoPandas documentation.
+Visualizing NYC Boroughs
+------------------------
 
-TODO: autogenerate these from docs themselves
+Visualize the Boroughs of New York City with Geopandas.
 
-Kelsey Jordahl
-Time-stamp: <Tue May  6 12:17:29 EDT 2014>
+This example generates many images that are used in the documentation. See
+the `Geometric Manipulations <geometric_manipulations>` example for more
+details.
+
+First we'll import a dataset containing each borough in New York City. We'll
+use the ``datasets`` module to handle this quickly.
 """
 import numpy as np
 import matplotlib.pyplot as plt
 from shapely.geometry import Point
 from geopandas import GeoSeries, GeoDataFrame
+import geopandas as gpd
 
 np.random.seed(1)
 DPI = 100
 
-# http://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nybb_16a.zip
-boros = GeoDataFrame.from_file('nybb.shp')
+path_nybb = gpd.datasets.get_path('nybb')
+boros = GeoDataFrame.from_file(path_nybb)
 boros.set_index('BoroCode', inplace=True)
-boros.sort()
-boros.plot()
+boros.sort_index()
```

Comment:
this line is not doing anything, as it is not inplace, so `boros = boros.sort_index()` (but not sure this sorting is actually needed)